### PR TITLE
Reset console color mode on Windows

### PR DESCRIPTION
**Pull request by zanderz**
I started trying to use the current mode as a default instead of assuming white on black, but it was too complicated. This will at least restore the user's colors.
@maxtaco @cjb

### CODE REVIEW

File: go/keybase/main.go [hunk 1]
```diff
@@ -115,6 +115,9 @@ func mainInner(g *libkb.GlobalContext) error {
 	checkSystemUser(g.Log)
 
 	if !cl.IsService() {
+		if err = logger.SaveConsoleMode(); err == nil {
```
Comment (maxtaco) [inline go/keybase/main.go]: This error will be overwritten below, is that intended?  It won't cause an exit out of this `mainInner`, which is unlike all of the other settings of `err`

Comment (zanderz) [inline go/keybase/main.go]: I thought it should only be used to gate the call to RestoreConsoleMode. Maybe I either don't need to use err, or could make one with a different name.

Comment (maxtaco) [inline go/keybase/main.go]: Yeah, that's what I thought too. Make the err locally scoped then?

**Comment by maxtaco** [comment]
Ok, :1

**Comment by maxtaco** [comment]
OK, 👍

**Pull request merged**